pkgs/development/python-modules/dynaconf/default.nix

```diff
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  ansible-core,
+}:
+
+buildPythonPackage rec {
+  pname = "dynaconf";
+  version = "3.2.10";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "dynaconf";
+    repo = "dynaconf";
+    tag = version;
+    hash = "sha256-xY5fCkhZP3nY2QnTtauLhbiEhUgANeavoxYsm5DsIFc=";
+  };
+
+  build-system = [ setuptools ];
+
+  dependencies = [ ansible-core ];
+
+  pythonImportsCheck = [ "dynaconf" ];
+
```

Comment:
This package has tests that should be run

https://github.com/dynaconf/dynaconf/tree/master/tests
https://github.com/dynaconf/dynaconf/blob/master/pyproject.toml#L54

Reply:
I think it would be as simple as

```nix
nativeCheckInputs = [
  pytestCheckHook
  pytest-mock
  radon
  flask
  /* remaining test dependencies */
];
```

and it is required for new Python packages, as far as I know.